lib/locale/pl/strings.php

```diff
@@ -35,6 +35,7 @@
 $_LANG['Other'] = 'Inny';
 $_LANG['Offer'] = 'Oferta';
 $_LANG['Fault'] = 'Usterka';
+$_LANG['Ticket deadline can be changed only by verifier if it is set'] = 'Termin ostateczny może zmienić jedynie weryfikator jeśli jest on ustawiony';
```

Comment:
Przeczytaj to jeszcze raz - zdanie brzmi bardzo dwuznacznie - nie wiadomo czy człon "jeśli ...." tyczy się terminu ostatecznego czy weryfikatora.